src/api.js

```diff
@@ -605,6 +627,52 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
         return res;
     };
 
+    /**
+     * @classdesc
+     * Represents an iterator over multiple SQL statements in a string,
+     * preparing and returning a Statement object for the next SQL
+     * statement on each iteration.
+     *
+     * You can't instantiate this class directly, you have to use a
+     * {@link Database} object in order to create a statement.
+     *
+     * **Warning**: When you close a database (using db.close()),
+     * using any statement iterators created by the database will
+     * result in undefined behavior.
```

Comment:
This is dangerous and confusing for the user. Can't we just throw an error if the database is closed ?

Reply:
Yes.

Reply:
@cpainterwakefield Has this been corrected ?

Reply:
It looks like it has been fixed. So we should remove the warning.

Reply:
Oops, yes, I will fix that.